### PR TITLE
First Pull Request

### PR DESCRIPTION
**Pull request by jimmyhurricanedang**
- Setup Frontend and Backend
- Connected Frontend to Google Maps Api
- Show a map from Google
- Show current location
- Pan to current location
- Adding belonging components
- Added list- and addMethod
=> tests doesn't work yet

### CODE REVIEW

**Review by lisamittmann — COMMENTED** [review]
looking good so far, unfortunately no tests, so no approval from me your friendly neighbourhood test nerd ;) 
- please check your frontend .gitignore, you currently have your node_modules included in your commit
- you have two package.json files, did you maybe do a npm install outside your frontend folder? please check your dependencies and if need be install all react packages in the correct folder ;)

File: backend/src/main/java/de/jimmy/columbus/controller/RouteController.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    private final RouteService routeService;
+
+    @Autowired
+    public RouteController(RouteService routeService){
+        this.routeService = routeService;
+    }
+
+    @GetMapping
+    public List<Route> listUsers() {
+        return routeService.listRoutes();
```
[the remark this answers is not in the document]
Comment (JimmyDang95) [reply backend/src/main/java/de/jimmy/columbus/controller/RouteController.java:25]: I did write two tests, but I reconstructed my whole model package in the backend which caused them to crash :( 

- Thought that it was fixed with Fabian but unfortunately it still doesn't work, gonna ask him tomorrow again !✌🏻
-  gonna check that ! thank u ! 

Didn't notice that ! Gonna rename it !

File: frontend/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM openjdk:15
+
+MAINTAINER REM Java 21 1 <jimmydang12@hotmail.de>
```
[the remark this answers is not in the document]
Comment (JimmyDang95) [reply frontend/Dockerfile:3]: Or maybe we should choose Fabian as Maintainer 🤪

File: backend/src/test/java/de/jimmy/columbus/service/RouteServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+class RouteServiceTest {
+
+    private final RoutesMongoDb routeDb = mock(RoutesMongoDb.class);
+    private final TimestampUtils timestampUtils = mock(TimestampUtils.class);
+
+
+    private final RouteService routeService = new RouteService(routeDb, timestampUtils);
+
+
```
[the remark this answers is not in the document]
Comment (JimmyDang95) [reply backend/src/test/java/de/jimmy/columbus/service/RouteServiceTest.java:30]: Spent the whole day writing these and they still don't work. Maybe someone wants to write them for me 🤷🏻‍♂️

File: backend/src/main/java/de/jimmy/columbus/utils/ParseUtils.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
+package de.jimmy.columbus.utils;
```
[the remark this answers is not in the document]
Comment (JimmyDang95) [reply backend/src/main/java/de/jimmy/columbus/utils/ParseUtils.java:1]: Yeah, but used it after reconstructing my whole model package✌🏻

**Pull request closed without merging**